### PR TITLE
fix(dedicated.pcc): add some tracking

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `fix/PCC-NSX-T` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9509 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added some tracking

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js [hunk 1]
```diff
@@ -149,18 +152,21 @@ export const US_GUIDES = [
   {
     title: 'dedicated_clouds_onboarding_us_guide1_title',
     description: 'dedicated_clouds_onboarding_us_guide1_description',
+    id: 'How-to-Access-the-NSX-Interface',
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js:155]: if duable, better to have same naming convention regarding others id.
i guess using lowercase id is better.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js [hunk 2]
```diff
@@ -149,18 +152,21 @@ export const US_GUIDES = [
...
     link:
       'https://support.us.ovhcloud.com/hc/en-us/articles/360015961459-How-to-Access-the-NSX-Interface',
   },
   {
     title: 'dedicated_clouds_onboarding_us_guide2_title',
     description: 'dedicated_clouds_onboarding_us_guide2_description',
+    id: 'How-to-Modify-the-vCenter-Access-Policy',
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js:162]: Related to :
https://github.com/ovh/manager/pull/8968/files#r1116740901

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js [hunk 3]
```diff
@@ -149,18 +152,21 @@ export const US_GUIDES = [
...
     link:
       'https://support.us.ovhcloud.com/hc/en-us/articles/1500007889561-How-to-Modify-the-vCenter-Access-Policy',
   },
   {
     title: 'dedicated_clouds_onboarding_us_guide3_title',
     description: 'dedicated_clouds_onboarding_us_guide3_description',
+    id: 'How-to-Authorize-IP-Addresses-for-vCenter-Access',
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js:169]: Related to :
https://github.com/ovh/manager/pull/8968/files#r1116740901

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js]: It's applied.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js]: It's applied.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js (hunk 3 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js]: It's applied.

**Review by mohammed-zahaf — APPROVED** [review]

**Review by y4nnL — APPROVED** [review]

**Pull request merged**